test/end-to-end/HACKING.md

```diff
@@ -0,0 +1,90 @@
+# Hacking on end to end test
+
+Here's where to get the code:
+
+    $ git clone https://github.com/weldr/cockpit-composer.git
+    $ cd test/end-to-end
+
+The remainder of the commands assume you're in the test/end-to-end folder.
+
+## Getting the development dependencies
+
+Cockpit-composer end to end test uses Node.js during case development.
+To make changes on Cockpit you'll want to install Node.js, NPM and
+various development dependencies.
+
+On Fedora:
+
+    $ sudo dnf install nodejs
+
+And lastly get development dependencies:
+
+    $ npm install
+
+## Preparing composer for testing on local machine
+
+It's easy to set up your local Linux machine for rapid deployment composer.
+First install `cockpit` and `lorax-composer` on your local machine:
+
+On Fefora:
+
+    $ sudo dnf install cockpit lorax-composer
+
+Next run this command from your top level Cockpit-composer checkout
+directory, and make sure to run it as the same user that you'll
+use to log into Cockpit below.
+
+    $ mkdir -p ~/.local/share/
+    $ ln -s ../../dist ~/.local/share/welder
```

Comment:
This is done by `make devel-install`.

Also, this just repeats what's already in README.md and this isn't really related to the end-to-end tests specifially, so IMHO this whole paragraph should stay in README.md.